java/client/src/org/openqa/selenium/support/events/EventFiringWebDriver.java

```diff
@@ -63,7 +64,7 @@
  * {@link WebDriverEventListener}, e&#46;g&#46; for logging purposes.
  */
 public class EventFiringWebDriver implements WebDriver, JavascriptExecutor, TakesScreenshot,
-    WrapsDriver, HasInputDevices, HasTouchScreen {
+    WrapsDriver, HasInputDevices, HasTouchScreen, Alert {
```

Comment:
@lukeis I see tests failed here: https://travis-ci.org/SeleniumHQ/selenium/jobs/173262596

Can you help me in resolving the same?